addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/QueuedSend.java

```diff
@@ -98,7 +98,7 @@ public void run() {
                     item = queue.take();
                 } catch (InterruptedException e) {
                     if (!willbeclosed) {
-                        logger.error("Queue take failed: " + e.getLocalizedMessage());
+                        logger.error("Queue take failed: {}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace